global.d.ts

```diff
@@ -0,0 +1,9 @@
+// global.d.ts
+import { CortexClient } from "./index";
+
+declare global {
+  // eslint-disable-next-line no-var
+  var testClient: CortexClient;
```

Comment:
Why not import this where needed? This will pollute the global scope for non-test code as well.